src/components/Cards/SummaryCard/SummaryCard.jsx

```diff
@@ -0,0 +1,44 @@
+import { useQuiz } from "contexts";
+import "./SummaryCard.css";
+
+export const SummaryCard = () => {
+	const { quizSummary } = useQuiz();
+	console.log(quizSummary, console.count());
+
+	const getOptionStyle = (ques, answer, option) => {
+		if (option === answer && answer === ques.answer) {
+			return "correct_answer";
+		} else if (option === answer) {
+			return "wrong_answer";
+		} else if (!!answer && option === ques.answer) {
+			return "correct_answer";
+		}
+	};
+
+	return (
+		<section className="summary__card">
+			{quizSummary?.questions.map((ques, index) => (
```

Comment:
What will happen if quizsummary is not undefined nor null value. so use a ternary condition instead

Reply:
But i haved defined it in context. so it won't render or it will render and the app won't break.